### ISSUE TITLE
Question regarding continuing of training

### ISSUE THREAD

**Issue by Quasimondo** [phillipi/pix2pix#40]
Using the "continue_train" option I can continue training a model. What I would like to know is if this is really continuing where the training was interrupted or more of a fine-tuning initialized from a previous model? The reason I ask is that whilst the previous model is loaded the epochs counter gets reset to 1 and I am not sure what happens with the learning rate and other parameters.

**Comment by phillipi** [comment]
It's more like finetuning. The learning parameters get reset. (note that the learning rate never changes in the current training code; you may want to change this to drop it after some number of epochs)

**Comment by Quasimondo** [comment]
Thanks for the clarification!
